scripts/prepare-dotenv.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
Should we document this in the README?

Reply:
is that really needed? it is used on `postinstall` which is transparent for developer.

Reply:
Anyway, it's barely explained on [README.md](https://github.com/pokt-network/pocketdex/pull/7/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R66)